includes/cancel-on-next-payment-date.php

```diff
@@ -251,10 +249,7 @@ function pmproarc_conpd_pmpro_email_body( $body, $email ) {
  *
  * @return array The updated email template variable data.
  */
-function pmproarc_conpd_pmpro_email_data( $data, $email ) {
-	if ( function_exists( 'pmproconpd_pmpro_email_data' ) ) {
-		return $data;
-	}
+function pmproconpd_pmpro_email_data( $data, $email ) {
```

Comment:
Check if this function already exists.

Reply:
Decision made in Slack that this check is not necessary.